MaskRCNN/model_rpn.py

```diff
@@ -10,16 +10,29 @@
 from config import config as cfg
 from model_box import clip_boxes
 from utils.mixed_precision import mixed_precision_scope
+from performance import print_buildtime_shape, print_runtime_shape, print_runtime_tensor
+import time
 
 @layer_register(log_shape=True)
 @auto_reuse_variable_scope
 def rpn_head(featuremap, channel, num_anchors, fp16=False):
     """
+    The RPN head that takes the feature map from the FPN and outputs bounding box logits.
+
+    For every pixel on the feature maps, there are a certain number of anchors.
+    The output will be:
+    label logits: indicate whether there is an object for a certain anchor in one pixel
+    box logits: The encoded box logits from fast-rccn paper https://arxiv.org/abs/1506.01497
+                page 5, in order to be consistent with the ground truth encoded boxes
+
+    Args:
+        featuremap: one feature map from FPN, i.e. P23456, BS x NA x H x W
+        channel: # channels of the feature map, scalar, default 256
+        num_anchors(NA): # of anchors for each pixel in the current feature map, scalar, default 3
```

Comment:
I would say 'for each location in the current feature map' since there is not an anchor on every pixel.